meetings/2023-05-11.md

```diff
@@ -0,0 +1,66 @@
+# Node.js  Security WorkGroup Meeting 2023-05-11
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=xtbjcbMjAvQ
+* **GitHub Issue**: https://github.com/nodejs/security-wg/issues/977
+* **Minutes Google Doc**: https://docs.google.com/document/d/19rq7F__F3qSdW8v3CeACJ6LpWHLlnWZsTgJWVWUwltI/edit
+
+## Present
+
+* Security wg team: @nodejs/security-wg
+Marco Ippolito @marco-ippolito
+Thomas GENTILHOMME @fraxken
+Ulises Gascon: @ulisesGascon
+
+
+## Agenda
```

Comment:
The list ends up on a single line otherwise:

```suggestion
## Present

* Security wg team: @nodejs/security-wg
  * Marco Ippolito @marco-ippolito
  * Thomas GENTILHOMME @fraxken
  * Ulises Gascon: @ulisesGascon

## Agenda
```